### PR TITLE
feat: improve-relay-service-requests

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/features/relay-service/use-relay-service-data-fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import {relayService} from '@docknetwork/wallet-sdk-core/lib/services/relay-service';
+import {captureException} from '@sentry/react-native';
+import {useCallback, useEffect, useState} from 'react';
+
+export async function fetchRelayServiceMessages({didDocs, onMessages}: any) {
+  const result = await relayService
```
Comment (mike-parkhill) [inline src/features/relay-service/use-relay-service-data-fetcher.ts:6]: assert onMessages?

File: src/features/relay-service/use-relay-service-data-fetcher.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      });
+    };
+
+    fetchTimeoutId = setTimeout(doFetchMessages, refreshRate);
+
+    return () => clearTimeout(fetchTimeoutId);
+  }, [fetchMessages, skipAutoFetch, refreshRate]);
+
+  const increaseRefreshRate = useCallback(() => {
+    setRefreshRate(fastRefreshRate);
```
[the remark this answers is not in the document]
Comment (maycon-mello) [reply src/features/relay-service/use-relay-service-data-fetcher.ts:69]: Not yet. Let me create a new event when the verification flow ends

[the remark this answers is not in the document]
Comment (maycon-mello) [reply src/features/relay-service/use-relay-service-data-fetcher.ts:80]: Done

File: src/features/relay-service/use-relay-service-data-fetcher.ts (hunk 1 above)
Comment (maycon-mello) [reply src/features/relay-service/use-relay-service-data-fetcher.ts:11]: Done

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**